application/F3DOptionsTools.cxx

```diff
@@ -214,21 +222,23 @@ void PrintHelp(const std::string& execName, const cxxopts::Options& cxxOptions)
   orderedCLIGroupNames.reserve(::CLIOptions.size());
   for (const ::CLIGroup& optionGroup : ::CLIOptions)
   {
-    // This ensure help is provided in the expected group order
     orderedCLIGroupNames.emplace_back(optionGroup.GroupName);
   }
+  
   f3d::log::info(cxxOptions.help(orderedCLIGroupNames));
+  
   f3d::log::info("Keys:");
   for (const auto& [key, desc] : f3d::interactor::getDefaultInteractionsInfo())
   {
-    ::PrintHelpPair(key, desc);
+    ::PrintHelpPair(key, desc, 20);
```

Comment:
why changing this ?

Reply:
The keys section looks more close to each other.

Reply:
can you share sceenshot of the result ?

Reply:
Yes most certainly. Here you go.
![IMG-20240918-WA0000.jpg](https://github.com/user-attachments/assets/20aa1394-75ad-4096-bbea-5956d7d7b46c)

When I increase the number the gap between texts grow wider.

Reply:
I dont see how this looks better than current version.

Reply:
Okay I won't tamper with the looks then I will implement the solution that you mentioned earlier. Thanks for putting up.